js/ludwig.js

```diff
@@ -11,7 +11,13 @@ class Ludwig {
 		this.branch = configuration.branch || 'master';
 		this.template = configuration.template;
 		this.prefix = configuration.prefix;
-		this.ludwigCreateSuggestionURL = configuration.ludwigCreateSuggestionURL;
+		const suggestionDomain = configuration.ludwigCreateSuggestionDomain? configuration.ludwigCreateSuggestionDomain : 'http://localhost:3000';
```

Comment:
Wouldn't it be more debuggable to fail if it is not set, and ensure the defaults are set in the config itself?